modules/server/src/main/scala/higherkindness/compendium/core/ProtocolUtils.scala

```diff
@@ -32,9 +33,12 @@ object ProtocolUtils {
   def impl[F[_]: Sync]: ProtocolUtils[F] = new ProtocolUtils[F] {
     override def validateProtocol(protocol: Protocol): F[Protocol] =
       if (protocol.raw.trim.isEmpty)
-        Sync[F].raiseError(new org.apache.avro.SchemaParseException("Protocol is empty"))
+        Sync[F].raiseError(SchemaParseException("Protocol is empty"))
       else
-        Sync[F].catchNonFatal(parser.parse(protocol.raw)).map(_ => protocol)
+        Sync[F]
```

Comment:
You might want to introduce https://github.com/augustjune/context-applied in the project, so you could just do `F.___`

Reply:
I didn't know about the existence of that plugin. However, the whole project is written in this way (which I like more...), so if we want to change this, I think we should also change the whole project. I'm not against this, but I think we can do it in another PR 🙂

Reply:
https://github.com/higherkindness/compendium/issues/165